hbase-server/src/main/java/org/apache/hadoop/hbase/LocalHBaseCluster.java

```diff
@@ -173,6 +173,14 @@ public LocalHBaseCluster(final Configuration conf, final int noMasters,
     for (int i = 0; i < noMasters; i++) {
       addMaster(new Configuration(conf), i);
     }
+
+    // Populate the master address host ports in the config. This is needed if a master based
+    // registry is configured for client metadata services (HBASE-18095)
+    List<String> masterHostPorts = new ArrayList<>();
+    getMasters().forEach(masterThread ->
```

Comment:
Java 8 idioms are just going to get in the way if you want to backport to branch-1, where Java 7 source level compliance is required.  FWIW

Reply:
Thanks for the heads up.